mplc/scenario.py

```diff
@@ -393,14 +412,12 @@ def copy(self, **kwargs):
                     'scenario_name',
                     'short_scenario_name',
                     'save_folder',
-                    'splitter']:
+                    'splitter',
+                    ]:
             del params[key]
         if 'is_quick_demo' in kwargs and kwargs['is_quick_demo'] != self.is_quick_demo:
             raise ValueError("Attribute 'is_quick_demo' cannot be modified between copies.")
-        if self.save_folder is not None:
-            params['save_path'] = self.save_folder.parents[0]
-        else:
-            params['save_path'] = None
+        params['save_path'] = self.save_folder.parents[0]
```

Comment:
`self.save_folder.parent` should do the same think and be a bit more concise.

Reply:
It was done already in one of the PRs recently merged - got it with the rebase.